src/Uno.UI/DirectUI/AccessKeyStringBuilder.cs

```diff
@@ -0,0 +1,87 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+// MUX Reference AccessKeyStringBuilder.cpp, tag winui3/release/1.5-stable
```

Comment:
The tag should be something like `1.5.x-stable`:

![image](https://github.com/unoplatform/uno/assets/1075116/8a18fa91-4277-4ad5-aacb-1b028e3b50d5)

Reply:
Ah, I see. I marked the branch, not the tag, thanks.